setup.cfg

```diff
@@ -134,6 +134,10 @@ all =
 include =
     astronomer.*
 
+[options.entry_points]
+apache_airflow_provider=
```

Comment:
Is the name correct or should we rename it to include astronomer?

Reply:
I don't think https://airflow.apache.org/docs/apache-airflow-providers/index.html#creating-your-own-providers